packages/state-manager/src/sagas/identity/registerCertificate/registerCertificate.saga.ts

```diff
@@ -22,16 +24,30 @@ export function* registerCertificateSaga(
   }
 
   if (currentCommunity.CA?.rootCertString) {
-    const payload: RegisterOwnerCertificatePayload = {
-      communityId: action.payload.communityId,
-      userCsr: action.payload.userCsr,
+    const identity = yield* select(identitySelectors.selectById(currentCommunity.id))
+    if (!identity?.userCsr || !currentCommunity?.rootCa) {
+      console.error("User CSR or root cert missing", identity?.userCsr, currentCommunity?.rootCa)
+      return
+    }
+
+    const payload: InitCommunityPayload = {
+      id: currentCommunity.id,
+      peerId: identity.peerId,
+      hiddenService: identity.hiddenService,
+      certs: {
+        // Hacking, perhaps make certs.certificate optional
+        certificate: identity.userCertificate || '',
```

Comment:
call it bikeshedding on my part but `||` is a logic operator, so there is problem with how it's typed
I highly recommend using `??` instead